tools/web_features/web_feature_map.py

```diff
@@ -0,0 +1,136 @@
+
+import itertools
+
+from collections import OrderedDict
+from os.path import basename
+from typing import Dict, List, Optional, Sequence, Set
+
+from ..manifest.item import ManifestItem, URLManifestItem
+from ..manifest.sourcefile import SourceFile
+from ..metadata.webfeatures.schema import FeatureEntry, FeatureFile, WebFeaturesFile
+
+
+class TestInfoForWebFeature:
+    """
+    Collection of extracted test information that relates to a given web feature
+    """
+    def __init__(self, manifest_item: URLManifestItem) -> None:
+        self.url = manifest_item.url
+
+    def to_dict(self) -> Dict[str, str]:
+        return {
+            "url": self.url
+        }
+
+
+class WebFeaturesMap:
+    """
+    Stores a mapping of web features to their associated test files.
```

Comment:
Judging by the unit tests it's a map to test paths, not file paths, right? Or what happens with `.any.js` tests?